packages/models/src/lib/persist-config.ts

```diff
@@ -1,12 +1,12 @@
 import { z } from 'zod';
 import { generalFilePathSchema } from './implementation/schemas';
 
+export const formatSchema = z.enum(['json', 'stdout', 'md']);
+export type Format = 'json' | 'stdout' | 'md';
```

Comment:
So that we only have one source of truth:

```suggestion
export type Format = z.infer<typeof formatSchema>;
```